test/01_basic/cibuildwheel_test.py

```diff
@@ -0,0 +1,50 @@
+import subprocess, sys, os
+from glob import glob
+project_dir = os.path.dirname(__file__)
+sys.path.append(os.path.join(os.path.dirname(project_dir), 'shared'))
+
+import utils
+
+def test():
+    # build the wheels
+    subprocess.check_call([sys.executable, '-m', 'cibuildwheel', project_dir]) 
+    
+    # check that every wheel is produced
+    if utils.platform == 'linux':
+        assert glob('wheelhouse/*-cp27-cp27m-manylinux1_x86_64.whl')
```

Comment:
> What about adding all filenames to utils (possibly in a function that takes project and version parameters, and then generates the filename),

ooh.... i'm maybe... 50/50 on this. I like the explicitness of how it's currently written, it's very clear what's being looked for and the logic is fully within the test. But a set comparison might be neater and we could run it for every test, not just these two. I might try an implementation and see how it feels

Reply:
Fair enough; just an idea. Indeed, if this works for the basic test and the tests on skipping, that should be enough, and the other tests can focus on something else. I was also just thinking of getting rid of the `--print-build-identifiers` dependency in the test, since there could be errors in there as well?